src/fetch/BaseFetcher.js

```diff
@@ -286,10 +286,15 @@ export const BaseFetcher = class {
       const min = new TagRemovingMinimizer(['style', 'script', 'meta', 'link']);
       const minDoc = await min.min(doc, { timer });
 
-      const hostname = (new URL(url)).hostname;
+      const baseUrl = new URL(url);
+      let hostname = baseUrl.hostname;
+      if (hostname === 'www.google.com' && baseUrl.pathname.startsWith('/maps')) {
+        hostname += '/maps';
```

Comment:
lets do this instead:
- make domainSpecific into an array
- do a "startsWith" check on the target URL

eg.
```
[
  { prefix: "https://x.com", instruction: "..." },
...
]
```

Reply:
prefix can be a regex actually